kibbeh/src/modules/settings/SoundEffectSettingsPage.tsx

```diff
@@ -57,7 +57,7 @@ export const SoundEffectSettings: React.FC<ChatSettingsProps> = () => {
 
         {Object.keys(soundEffectSettings).map((k) => {
           return (
-            <div className={`flex mb-4 items-center`} key={k}>
+            <div className={`flex flex mb-4 items-center`} key={k}>
```

Comment:
I don't understand why you change things like this that are pointless and aren't related to the pr

Reply:
@ajmnz uhh vscode tells me 2 flex are redundant and makes the file yellow, so i did that

Reply:
wait did i add it or remove it?

Reply:
i was supposed to remove it, but it says i added it. oh i see what happened, i had copied the whole kibbeh folder from my prev PR and pasted it here, git now thinks i added that

Reply:
there are so many redundant flex that i had removed but those are lost somewhere

Reply:
ive added it back now